main/main.c

```diff
@@ -554,6 +554,14 @@ void interactiveLoop (cookedArgs *args CTAGS_ATTR_UNUSED, void *user CTAGS_ATTR_
 			openTagFile ();
 			if (size == -1)
 			{					/* read from disk */
+#ifdef HAVE_SECCOMP
+				struct interactiveModeArgs *iargs = user;
+				if (iargs->secure) {
+					error (FATAL,
+						   "invalid request in secure submode: reading file contents from a file is limited");
+					goto next;
```

Comment:
Well, why do we have documents then? :-)

for testing --_secure, you need a code path that tries to do something illegal. I think we don't want to add a buffer overflow to ctags, so how else do you want to test it, if not this code path?